rskj-core/src/main/java/co/rsk/peg/federation/FederationSupportImpl.java

```diff
@@ -481,25 +483,18 @@ public int voteFederationChange(Transaction tx, ABICallSpec callSpec, SignatureC
         return (int) result.getResult();
     }
 
-    private boolean unknownFederationChangeFunction(String calledFunction) {
-        return Arrays.stream(FederationChangeFunction.values()).noneMatch(fedChangeFunction -> fedChangeFunction.getKey().equals(calledFunction));
-    }
-
-    private ABICallVoteResult executeVoteFederationChangeFunction(boolean dryRun, ABICallSpec callSpec, BridgeEventLogger eventLogger) throws BridgeIllegalArgumentException {
+    private ABICallVoteResult executeVoteFederationChangeFunction(boolean dryRun, byte[][] callSpecArguments, FederationChangeFunction federationChangeFunction, BridgeEventLogger eventLogger) throws BridgeIllegalArgumentException {
         // Try to do a dry-run and only register the vote if the
         // call would be successful
```

Comment:
Not sure this comment belongs here, the dryRun is a parameter. It's on `voteFederationChange` function where the dryRun is first attempted